docs/DeveloperGuide.md

```diff
@@ -389,12 +391,12 @@ For more information, see [Implementation for Assign/Unassign Feature](#assignun
 To increase the convenience of use for our expected typist user, we introduced a few mass operations related to leave:
 
 - `TakeLeaveCommand` and `CancelLeaveCommand` both allow for many worker to one shift leave assignment similar to 
-`AssignCommand` and `UnassignCommand`.
-- `MassTakeLeaveCommand` and `MassCancelLeaveCommand` allow for one worker to many shift leave assignment.
-  - These two mass commands allow for taking leave over a range of dates - similar to how leave is often planned 
+`AssignCommand` and `UnassignCommand`. See [MassOps Implementation Details](#massops-feature).
+- `MassTakeLeaveCommand` and `MassCancelLeaveCommand` allow for one worker to many shift-leave assignment.
+  - These two mass commands allow for taking leave over a range of days - similar to how leave is often planned 
   by workers.
-  - These two commands do not require `Shift`s representing the datetime range to take leave to be present. A bare-bones 
-  `Shift` with only `ShiftDate` and `ShiftTime` will be initialised for each `Shift` that has no identity equivalent 
+  - These two commands do not require `Shift`s representing the daytime range to take leave to be present. A bare-bones 
```

Comment:
```suggestion
  - These two commands do not require `Shift`s representing the daytime range to take leave to be present. A bare-bones 
```
daytime might be a little confusing but I get what you mean

Reply:
Editing to make it clearer